recipes/qt/5.x.x/conanfile.py

```diff
@@ -368,8 +368,7 @@ def requirements(self):
             self.requires("libalsa/1.2.5.1")
         if self.options.gui and self.settings.os in ["Linux", "FreeBSD"]:
             self.requires("xorg/system")
-            if not tools.cross_building(self, skip_x64_x86=True):
-                self.requires("xkbcommon/1.3.1")
+            self.requires("xkbcommon/1.3.1")
```

Comment:
it's already broken since xkbcommon is missing, while it's required.

Reply:
It's been a long time since I tested it, but it worked at least partially. Requirements can depend on settings, so I don't get why tools.cross_building would be forbidden in `requirements(self)`

Reply:
Because requirements are host dependencies, they don't care of build machine, what does would it mean to not require a dependency if cross-building, then require it at consume time if someone use it on its machine?

Reply:
I think you have reached the agreement, as Eric approved it, so I am comfortable approving it as well (I am not very much in Qt nowadays, but from the generic CCI perspective it looks fine)